packages/react-devtools-timeline/src/TimelineContext.js

```diff
@@ -8,16 +8,19 @@
  */
 
 import * as React from 'react';
-import {createContext, useCallback, useMemo, useState} from 'react';
-import createDataResourceFromImportedFile from './createDataResourceFromImportedFile';
+import {createContext, useMemo, useRef, useState} from 'react';
 
-import type {HorizontalScrollStateChangeCallback, ViewState} from './types';
-import type {DataResource} from './createDataResourceFromImportedFile';
+import type {
+  HorizontalScrollStateChangeCallback,
+  SearchRegExpStateChangeCallback,
+  ViewState,
+} from './types';
+import type {RefObject} from 'shared/ReactTypes';
 
 export type Context = {|
-  clearTimelineData: () => void,
-  importTimelineData: (file: File) => void,
-  timelineData: DataResource | null,
+  file: File | null,
```

Comment:
why did we need to refactor this to rename this to `file` vs the old `timelineData` and all the other changes that trickle down from this? just curious

Reply:
This refactor was to make it work more like our other suspense caches. Maybe we can chat about this offline?